src/main/java/view/SignupView.java

```diff
@@ -39,40 +39,116 @@ public SignupView(SignupController controller, SignupViewModel signupViewModel)
+        usernameInputField.setForeground(Color.GRAY);
+        usernameInputField.setText("e.g. dennis_ivy");
+        JPanel usernamePanel = new JPanel();
+        usernamePanel.setLayout(new BoxLayout(usernamePanel, BoxLayout.Y_AXIS));
+        usernamePanel.setBackground(new Color(50, 50, 60));
+        usernamePanel.add(usernameInfo);
+        usernamePanel.add(usernameInputField);
+
+        // password  panel
+        JLabel passwordInfo = new JLabel(SignupViewModel.PASSWORD_LABEL);
+        passwordInfo.setForeground(Color.WHITE);
+        passwordInputField.setBackground(new Color(50, 50, 60));
+        passwordInputField.setForeground(Color.blue);
+        JPanel passwordPanel = new JPanel();
+        passwordPanel.setLayout(new BoxLayout(passwordPanel, BoxLayout.Y_AXIS));
+        passwordPanel.setBackground(new Color(50, 50, 60));
+        passwordPanel.add(passwordInfo);
+        passwordPanel.add(passwordInputField);
+
+
+        // repeat password panel
+        JLabel repeatPasswordInfo = new JLabel(SignupViewModel.REPEAT_PASSWORD_LABEL);
+        repeatPasswordInfo.setForeground(Color.WHITE);
+        repeatPasswordInputField.setBackground(new Color(50, 50, 60));
+        repeatPasswordInputField.setForeground(Color.blue);
+        JPanel repeatPasswordPanel = new JPanel();
+        repeatPasswordPanel.setLayout(new BoxLayout(repeatPasswordPanel, BoxLayout.Y_AXIS));
+        repeatPasswordPanel.setBackground(new Color(50, 50, 60));
+        repeatPasswordPanel.add(repeatPasswordInfo);
+        repeatPasswordPanel.add(repeatPasswordInputField);
+
+        // SIGN UP TITLE:
+
+        JPanel headerPanel = new JPanel();
+        headerPanel.setBackground(new Color(19, 19, 223));
+        JLabel headerLabel = new JLabel(SignupViewModel.TITLE_LABEL);
+        headerLabel.setForeground(Color.WHITE);
+        headerLabel.setFont(new Font("Arial", Font.BOLD, 12));
+        headerPanel.add(headerLabel);
+
+        // sign up button
+        signUp = new JButton("SignUp");
+        signUp.setBackground(new Color(0, 123, 255));
+        signUp.setForeground(Color.BLUE);
+        signUp.setAlignmentX(Component.CENTER_ALIGNMENT);
+
+
+        // switch to login
         JLabel signedUpLabel = new JLabel("Already signed up?");
+        signedUpLabel.setForeground(Color.LIGHT_GRAY);
+        signedUpLabel.setAlignmentX(Component.CENTER_ALIGNMENT);
+
         toLogin = new JButton(SignupViewModel.TO_LOGIN_BUTTON_LABEL);
+        toLogin.setCursor(Cursor.getPredefinedCursor(Cursor.HAND_CURSOR));
+        toLogin.setForeground(Color.BLUE);
 
-        panel.add(signedUpLabel);
-        panel.add(toLogin);
-        buttons.add(panel);
+        // cancel button
 
         cancel = new JButton(SignupViewModel.CANCEL_BUTTON_LABEL);
-        buttons.add(cancel);
+        cancel.setForeground(Color.BLUE);
+
+        // Create the label
+
+
+        // TODO: loginLink.setCurosr - what is that?
```

Comment:
it specifies what type of mouse cursor should appear when it hovers over a specific the component. I don't think it's necessary.

Reply:
thanks! added that too?